CMakeLists.txt

```diff
@@ -216,6 +216,7 @@ option(CHIP_ENABLE_NON_COMPLIANT_DEVICELIB_CODE "Enable non-compliant devicelib
 option(CHIP_FAST_MATH "Use native_ OpenCL functions which are fast but their precision is implementation defined" OFF)
 option(CHIP_USE_INTEL_USM "enable support for cl_intel_unified_shared_memory in the OpenCL backend" OFF)
 option(CHIP_EXT_FLOAT_ATOMICS "Use cl_ext_float_atomics." ON)
+option(CHIP_L0_FIRST_TOUCH "First-touch workaroudn for Level Zero." ON)
```

Comment:
```suggestion
option(CHIP_L0_FIRST_TOUCH "First-touch workaround for Level Zero." ON)
```

Reply:
Can you link to the issue in comments of this flag? "First-touch workaround" might not say much to outside users nor to us in a few years time.